lib/runAll.js

```diff
@@ -79,6 +79,14 @@ export const runAll = async (
 ) => {
   debugLog('Running all linter scripts')
 
+  const hasExplicitCwd = !!cwd
+  if (hasExplicitCwd) {
+    debugLog('Using explicit cwd `%s` from options', cwd)
+  } else {
+    cwd = process.cwd()
+    debugLog('Using default cwd `%s`', cwd)
+  }
+
```

Comment:
Maybe consider using only `debug` and keep the default parameter? Is there a value in having two code paths here? The only reason I see is to inform about "explicit" but I'm not sure why it's so important. Just printing the `cwd` would not be enough?

Reply:
I can use the `hasExplicitCwd` later in the multiple config feature.

Reply:
I updated this a bit. I still left separate logging, because in the next PR I want to use the knowledge of whether the cwd was explicitly set or not.